.circleci/config.yml

```diff
@@ -179,7 +186,7 @@ workflows:
             branches:
               only: development
       - deploy-to-development:
-          # context: api-assume-role-housing-development-context
```

Comment:
Please, remove this comment if it's not needed

Reply:
it is not a comment but git way of showing the line that was removed